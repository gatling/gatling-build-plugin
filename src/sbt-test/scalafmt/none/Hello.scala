object Hello extends App { // empty line after declaration NOT appended
  final implicit val a: Long = 0L
  val coucou: Int = 0
}
