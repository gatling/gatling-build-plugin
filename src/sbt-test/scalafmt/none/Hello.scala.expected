object Hello extends App { // empty line after declaration NOT appended
  final implicit val a: Long = 0L
  val coucou: Int = { 1 } // do NOT remove curly braces
}
